recipes/fortran-binary/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "fortran-binary" %}
+{% set version = "1.0.6" %}
+{% set sha256 = "be613276fb3141b81933c508990b8fa5e71f1df04a7423567d2aa78a16267241" %}
```

Comment:
Can you actually undo the templating of the sha and build? It's not usually done any more like that and can make things more difficult for the auto-update bot.